framework/packages/abstract-sdk/src/base/endpoints/modules_ibc.rs

```diff
@@ -16,6 +17,19 @@ pub trait ModuleIbcEndpoint: Handler {
         info: MessageInfo,
         msg: ModuleIbcMsg,
     ) -> Result<Response, Self::Error> {
+        // Make sure module have ibc_client as dependency
+        if !self
```

Comment:
When a module receives an IBC message it isn't required to have a dependency on the IBC client (the message is sent from the host)